src/v2i-hub/CARMACloudPlugin/src/CARMACloudPlugin.cpp

```diff
@@ -229,6 +229,19 @@ void CARMACloudPlugin::CARMAResponseHandler(QHttpEngine::Socket *socket)
 	container.load<XML>(ss);
 	tsm5message.set_contents(container.get_storage().get_tree());
 	tsm5ENC.encode_j2735_message(tsm5message);
+	string enc = tsm5ENC.get_encoding();

```

Comment:
This code is duplicated from the other Broadcast_TCMs method, can you pull it out into a separate method and call it from both places to avoid duplication?